socorro/unittest/external/es/test_crashstorage.py

```diff
@@ -98,24 +97,7 @@
     },
 }
 
-a_processed_crash_with_no_stackwalker = deepcopy(a_processed_crash)
```

Comment:
I changed the tests that used this variable so that they build the processed crash as part of the test.